src/plugins/janus_videoroom.c

```diff
@@ -3168,6 +3176,8 @@ static json_t *janus_videoroom_subscriber_streams_summary(janus_videoroom_subscr
 		json_object_set_new(m, "active", (ps || stream->type == JANUS_VIDEOROOM_MEDIA_DATA) ? json_true() : json_false());
 		json_object_set_new(m, "mindex", json_integer(stream->mindex));
 		json_object_set_new(m, "mid", json_string(stream->mid));
+		if(stream->opaqueid)
+			json_object_set_new(m, "opaque_id", json_string(stream->opaqueid));
```

Comment:
I'm not sure this should always be here. `janus_videoroom_subscriber_streams_summary` is used for different targets, not just notifying updates back to the subscriber, and this opaque identifier feels like something that only belongs to that subscriber's events.

Reply:
I see the place actually exactly here. This is the element in the streams list i cannot associated if i don´t have the cross reference id with the subscribe event. Thus this is the idea of the PR to have the correlation id in the streams list as provided in the subscribe request.